app/assets/stylesheets/admin.scss

```diff
@@ -18,7 +18,20 @@ $proxima-nova-family: "Proxima Nova", 'Helvetica Neue', Helvetica, Arial, 'Lucid
     @include no-bottom-border;
     padding-left: 6em;
 
-    .label { display: none; }
+    .label {
+      display: none;
+    }
+  }
+}
+
+.js_editor_panel,
```

Comment:
Selector `js_editor_panel` should be written in lowercase with hyphens<br>Selector `css_editor_panel` should be written in lowercase with hyphens